Source/Resource/boolean_profiles/default.json

```diff
@@ -0,0 +1,775 @@
+{
+    "conditions": [
+        {
+            "group": 0,
```

Comment:
oh i get that there's a link to the `plotGroup` value but isn't this evident from the nested structure?

Reply:
yes, this is plotGroup. And yes, at the moment it is necessary as it is used (together with type) to identify plot. I was thinking about using name instead of group/type, but wasn't sure if names from different group/type can be duplicated